src/Application.js

```diff
@@ -23,13 +23,13 @@ export class Application {
      */
     this.state = new StateManager();
     this.state.addField('ready', false);
-    this.state.addField('soundMuted', false);
-    this.state.addField('captionsMuted', true);
-    this.state.addField('musicMuted', false);
-    this.state.addField('voMuted', false);
-    this.state.addField('sfxMuted', false);
     this.state.addField('pause', false);
+    this.state.addField('soundVolume', 1);
+    this.state.addField('musicVolume', 1);
+    this.state.addField('voVolume', 1);
+    this.state.addField('sfxVolume', 1);
     this.state.addField('playOptions', {});
+    this.state.addField('captions', true);
```

Comment:
So... I don't want to remove those old fields (and rename the captions field) because we're now going to break compatability with the SpringRollContainer v1.